Mod2/Assessments/Week2Assessment.md

```diff
@@ -0,0 +1,62 @@
+# Week 2 Assessment
+
+## Questions
+1. What does TDD stand for?
+
+1. What are three benefits of using TDD?
+
+1. Imagine you are in an interview.  The interviewer asks: How do you use TDD?
```

Comment:
I would add "How would you answer" here so it's extra clear what the student should do.